server/notification-providers/dingding.js

```diff
@@ -29,6 +33,9 @@ class DingDing extends NotificationProvider {
                     msgtype: "text",
                     text: {
                         content: msg
+                    },
+                    "at": {
+                        "isAtAll": notification.isAtAll // Here the isAtAll value in the notification object is used
```

Comment:
comment does not add valuable context => remove

```suggestion
                        "isAtAll": notification.isAtAll,
```